### PR TITLE
Update deps. Fix endpoint

### PR DESCRIPTION
**Pull request by kkuzmin**
### Problem Description
Sometimes customers input endpoint with `http` prefix

### Solution Description
Trim endpoint to domain name.
Update deps.

### CODE REVIEW

File: collectors/auth0/auth0_collector.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ const tsPaths = [
     { path: ['date'] }
 ];
 
+const HOSTNAME_REGEXP = /^[https]*:\/\/|\/$/g;
```
Comment (mtrybulec) [inline collectors/auth0/auth0_collector.js]: Why repeat the t here? A bit confusing...

Comment (kkuzmin) [reply collectors/auth0/auth0_collector.js]: I left it for readability to highlight that we'd like to remove `http/https` prefixes.
Do you think it is confusing?

Comment (mtrybulec) [reply collectors/auth0/auth0_collector.js]: :-) I guess it depends on the reader

File: collectors/auth0/auth0_collector.js [hunk 2]
```diff
@@ -37,8 +39,9 @@ class Auth0Collector extends PawsCollector {
 
     pawsGetLogs(state, callback) {
         let collector = this;
+        const hostname = process.env.paws_endpoint.toLowerCase().replace(HOSTNAME_REGEXP, '');
```
Comment (mtrybulec) [inline collectors/auth0/auth0_collector.js]: Hmm... might break if auth0 is case sensitive on endpoints; perhaps better to add i to the regex replace?

**Pull request merged**